docs/en/docs/insight/quickstart/install/component-scheduling.md

```diff
@@ -0,0 +1,321 @@
+# Custom Insight Component Scheduling Policy
+
+When deploying Insight to a Kubernetes environment, proper resource management and optimization are crucial.
+Insight includes several core components such as Prometheus, OpenTelemetry, FluentBit, Vector, and Elasticsearch.
+These components, during their operation, may negatively impact the performance of other Pods within the cluster
+due to resource consumption issues. To effectively manage resources and optimize cluster operations,
+node affinity becomes an important option.
+
+[taints](#configure-dedicated-nodes-for-insight-using-taints)
```

Comment:
This page is about how to add [taints](#configure-dedicated-nodes-for-insight-using-taints)